.Lib9c.Tests/Action/DailyRewardTest2.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Lib9c.Tests.Action
+{
+    using System.Linq;
+    using Libplanet;
+    using Libplanet.Action;
+    using Libplanet.Crypto;
+    using Nekoyume;
+    using Nekoyume.Action;
+    using Nekoyume.Model.Mail;
+    using Nekoyume.Model.State;
+    using Serilog;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class DailyRewardTest2
+    {
+        private readonly IAccountStateDelta _initialState;
+        private readonly Address _agentAddress;
+        private readonly Address _avatarAddress;
+
+        public DailyRewardTest2(ITestOutputHelper outputHelper)
+        {
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Verbose()
+                .WriteTo.TestOutput(outputHelper)
+                .CreateLogger();
+
+            _initialState = new State();
+            var sheets = TableSheetsImporter.ImportSheets();
+            foreach (var (key, value) in sheets)
+            {
+                _initialState = _initialState
+                    .SetState(Addresses.TableSheet.Derive(key), value.Serialize());
+            }
+
+            var tableSheets = new TableSheets(sheets);
+
+            _agentAddress = new PrivateKey().ToAddress();
+            var agentState = new AgentState(_agentAddress);
+            _avatarAddress = new PrivateKey().ToAddress();
+            var rankingMapAddress = new PrivateKey().ToAddress();
+            var avatarState = new AvatarState(
+                _avatarAddress,
+                _agentAddress,
+                0,
+                tableSheets.GetAvatarSheets(),
+                new GameConfigState(),
+                rankingMapAddress)
+            {
+                actionPoint = 0,
+            };
+            agentState.avatarAddresses[0] = _avatarAddress;
+
+            _initialState = _initialState
+                .SetState(Addresses.GameConfig, new GameConfigState().Serialize())
+                .SetState(_agentAddress, agentState.Serialize())
+                .SetState(_avatarAddress, avatarState.Serialize());
+        }
+
+        [Theory]
+        [InlineData(400000, 10)]
+        public void Execute(int itemId, int itemCount)
```

Comment:
The numbers, 400000 and 10, seem like the hard-coded values in `DailyReward2`. Therefore it seems not configurable with parameters. And I have wondered why this test uses `Theory`, it seems enough to use `Fact` 🤔 ? How do you think about this? 🙏

Reply:
> The numbers, 400000 and 10, seem like the hard-coded values in `DailyReward2`. Therefore it seems not configurable with parameters. And I have wondered why this test uses `Theory`, it seems enough to use `Fact` 🤔 ? How do you think about this? 🙏

I used test 'theory' with the assumption that the hardcoded values ​​will be modified later.
But when I think about it again.. :thinking_face:
Now it seems correct to use 'fact' and then use 'theory' when the hardcoding is modified (probably v3).